subsys/bluetooth/mesh/Kconfig

```diff
@@ -380,6 +380,17 @@ endchoice
 
 if BT_MESH_ADV_EXT
 
+config BT_MESH_ADV_MAIN_RELAY_ENABLED
+	bool "Advertiser main support relay messages"
+	default y if !BT_MESH_FRIEND && !BT_MESH_LOW_POWER
+	depends on BT_MESH_RELAY
+	help
+	  Enable advertiser main to support relay messages, this maximizes the
+	  utilization efficiency of advertising sets, which is helpful for the
+	  sending of dense relays, but for Friend and LPN, it delays the sending
+	  of local messages a little, this may cause the time slot deviation between
+	  friend and lpn, and the power consumption of lpn will increase.
```

Comment:
```suggestion
	  Enable advertiser main to support relay messages, this maximizes the
	  utilization efficiency of advertising sets, which is helpful for the
	  sending of dense relays, but for Friend and LPN, it delays the sending
	  of local messages a little, this may cause the time slot deviation between
	  friend and lpn, and the power consumption of lpn will increase.
```